src/ResourceManager/EventHub/Commands.EventHub/help/Remove-AzureRmEventHub.md

```diff
@@ -29,13 +42,46 @@ PS C:\> Remove-AzureRmEventHub -ResourceGroupName MyResourceGroupName -Namespace
 
 Removes the Event Hub \`MyEventHubName\`.
 
+### Example 2 - InputObject
+```
+PS C:\> Remove-AzureRmEventHub -InputObject $inputobject
+```
+
+Removes the Event Hub provided through $inputobject for -InputObject parameter.
+
+### Example 3 - ResourceId
+```
+PS C:\> Remove-AzureRmEventHub -ResourceId $resourceid
```

Comment:
@v-Ajnava [same comment as the ServiceBus PR](https://github.com/Azure/azure-powershell/pull/6732#discussion_r204583390) -- for examples using `-ResourceId`, please make sure to show the user where to get the value for the parameter

Reply:
resolved, updated examples with Piping scenario for namespace, variable and string scenario.